core/src/test/java/org/bitcoinj/utils/BigIntegerConversion.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2017 Nicola Atzei
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bitcoinj.utils;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+
+import org.bitcoinj.core.Utils;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class BigIntegerConversion {
```

Comment:
Although not strictly necessary, the convention for the naming of the test classes is `<class to test>Test`.